README.rst

```diff
@@ -82,6 +88,40 @@ The next step is to execute the leapfrog upgrade script and follow the prompts:
     scripts/ubuntu14-leapfrog.sh
 
 
+Executing a major upgrade
+----------------------------
+
+The first step is to checkout the rpc-upgrades repo.
+
+.. code-block:: shell
+
+    git clone https://github.com/rcbops/rpc-upgrades.git /opt/rpc-upgrades
+
+
+By default Elasticsearch data will be kept and Elasticsearch will be upgraded at the end of the leapfrog.
+If you'd like to reset the Elasticsearch data, you can override the upgrade and remove the container during
+the upgrade by setting these environment variables:
+
+.. code-block:: shell
+
+    export UPGRADE_ELASTICSEARCH="no"
+    export CONTAINERS_TO_DESTROY='all_containers:!galera_all:!neutron_agent:!ceph_all:!rsyslog_all'
+
+
+**Note:** *Currently the rpc-upgrades repo targets r14.11.0.  If you want to deploy the previous version you can:*
+
+.. code-block:: shell
+
+   export RPC_TARGET_CHECKOUT=r14.10.0
```

Comment:
This should bw r14.11.0 at least

Reply:
It's set to r14.10.0 since it's an example of "If you want to deploy the previous version you can:"